ci-operator/config/terraform-redhat/terraform-provider-red-hat-cloud-services/terraform-redhat-terraform-provider-red-hat-cloud-services-release-4.14.yaml

```diff
@@ -31,4 +31,4 @@ tests:
 zz_generated_metadata:
   branch: release-4.14
   org: terraform-redhat
-  repo: terraform-provider-ocm
+  repo: terraform-provider-red-hat-cloud-services
```

Comment:
not in the scope of this PR, but I wonder why we even have the configs for those ocp branches (4.14/4.15/etc)